src/services/cherry-picker.ts

```diff
@@ -398,7 +397,7 @@ export class CherryPicker {
       errorLog = '0'
     }
 
-    failure = failureLog === 'true' || parseInt(errorLog) > 5
+    failure = failureLog === 'true' || parseInt(errorLog) > 50
```

Comment:
I think this could be better as a constant like `FAILURE_RATE_THRESHOLD`

Reply:
Agree, but that holds true for ALL of the things I adjusted with this PR. Not sure I'm ready to commit to making them all variables. Sometimes that makes the code harder to read as you have to reference another file or definitely another spot in the code. For numbers only used once, I think this is tolerable, eh?